charts/datadog-operator/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: datadog-crds
   repository: file://../datadog-crds/
-  version: 1.0.1
-digest: sha256:f8c3481300e9d38be6466cefa03ff28d57a85dbdc957e7f42faefe74095fc9df
-generated: "2023-09-28T12:33:31.778506-04:00"
+  version: 1.3.0
+digest: sha256:86af5339c70e484401b7d340b2380802c93f946ceb0441d7348f74811917be8b
+generated: "2024-01-04T12:16:01.40644-05:00"
```

Comment:
This got updated because I updated and re-built the operator chart dependencies. Should I copy the sha256 and timestamp from the `datadog-operator-v1.4.1.` tag? I'm concerned this change might trigger a conflicting `datadog-operator-v1.4.1` chart tag.

Reply:
In the add-on environment chart dependencies should be packaged together with the wrapper, they aren't pulled from the Helm repo. So this should be fine. As long as chart can be rendered and installed properly we should be good.